src/Strut.hh

```diff
@@ -23,29 +23,44 @@
 #ifndef STRUT_HH
 #define STRUT_HH
 
+struct StrutDimensions {
+    int left, right, top, bottom;
+    StrutDimensions(int l, int r, int t, int b):
+        left(l), right(r), top(t), bottom(b) {}
+    void max_by(const StrutDimensions& dims) {
```

Comment:
expandTo? growTo?

Reply:
OK, I will try to choose a more felicitous name in the re-do. Thanks.

Reply:
Or it will be moot because to reduce refactoring I won't break out the strut dimensions into a struct this time.